<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileProtocolPacketSelection</class>
 <widget class="QWidget" name="FileProtocolPacketSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>26</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileProtocolPacketSelection</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Packets total:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_packetCountLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="_packetSelectionSlider">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="_packetSelectionSpinBox">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
